### PR TITLE
Feature/74 counter component

### PR DESCRIPTION
**Pull request by shawn320**
# Description 📝

Counter component that displays the number of benefits

counter

## Taiga Link 🔗

https://taiga.dts-stn.com/project/pond-dts-experimental-benefit-finder/us/74?milestone=34

## Checklist ✅
- [x] created story for all visual components
- [x] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
-    "tailwindcss": "^1.8.12"
+    "tailwindcss": "^1.9.5"
```
Comment (Moro-Code) [inline package.json:30]: just make sure to include the PR description that you bumped up dependencies. Also make sure to post on slack that you're doing so

**Review by Moro-Code — APPROVED** [review]
Once latest commit is signed it's good to go for me ! Thanks for the work 🚀

File: package.json (hunk 1 above)
Comment (shawn320) [reply package.json:30]: Thanks for point it out. I was not aware of this. I was messing with tailwind, tried to customize some style, seems like I updated the version...

**Review by MarcoGoC — APPROVED** [review]
good job shawn

**Pull request merged**